netbench/netbench/src/collector.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Question: Are we sure that we want to move this into the netbench crate? I'm not 100% sure on this, but if the netbench crate is mostly "netbench protocol" stuff, it might make sense to keep the "netbench-collector" stuff in it's own crate, and just have netbench-player take a dependency on that crate.

Reply:
I'm open to suggestions here.

Reply:
We currently use hyphens for [other crate names](https://github.com/aws/s2n-quic/blob/main/quic/s2n-quic-core/Cargo.toml#L2). It might just be the rust funkiness where you refer to the import path of `netbench-collector` as `netbench_collector`. At least I think that's how it goes, I frequently have to look it up.